win32ss/gdi/gdi32/CMakeLists.txt

```diff
@@ -64,6 +65,6 @@ target_link_libraries(gdi32
     atan2
     ${PSEH_LIB})
 
-add_importlibs(gdi32 user32 advapi32 kernel32 ntdll)
+add_importlibs(gdi32 usp10 user32 advapi32 kernel32 ntdll)
```

Comment:
Perhaps as a delay-load, or via a LoadLibrary call inside gdi32.

Reply:
Delayloads not until binutils is fixed.